### DISCUSSION TITLE
Nsight

### DISCUSSION THREAD

**Discussion by yrodriguez31** [OE4T/meta-tegra#1563] [General]
Hello,

Does anyone know how to use Nsight on a Jetson Xavier NX with an OS generated with the meta-tegra branch kirkstone_r32.7.x on an Ubuntu 22.04 host? I've tried to connect remotely to the host, but it's impossible because the Nsight version on the remote is too old. Even generating the Nsight report on the targer and trying to import it on my host results in the same problem. I don't have any graphical interface on the target.

Thanks

**Comment by ichergui** [comment]
Hi @yrodriguez31 
I'm not sure why you are saying too old. All our recipes are aligned with the appropriate JetPack/L4T release.
No need to have the GUI on the target. You can profile the application you are working on then you can `scp` the result file to your host machine and at this point you can visualise the result.
Kindly check the Nvidia online documentation
https://docs.nvidia.com/nsight-systems/2021.5/UserGuide/index.html

**Reply by yrodriguez31** [reply]
Hi,
 There is no need to have the same version of the nsight on the host and the remote ?

**Reply by ichergui** [reply]
They should be backward compatible. you can use recent version of nsight on your host machine.
